tests/test_pck_transformer.py

```diff
@@ -20,6 +20,28 @@ def test_get_cs_form_id_passes(self):
 
         self.assertEqual(form_id, 'Q01B', msg=None)
 
+        # QCAS
+        survey = {'survey_id': '019'}
+        response = {'collection': {'instrument_id': '0018'}}
+        pck_transformer = PCKTransformer(survey, response)
+        form_id = pck_transformer.get_cs_form_id()
+
+        self.assertEqual(form_id, '0018', msg=None)
+
+        survey = {'survey_id': '019'}
+        response = {'collection': {'instrument_id': '0019'}}
+        pck_transformer = PCKTransformer(survey, response)
+        form_id = pck_transformer.get_cs_form_id()
+
+        self.assertEqual(form_id, '0019', msg=None)
+
+        survey = {'survey_id': '019'}
+        response = {'collection': {'instrument_id': '0020'}}
+        pck_transformer = PCKTransformer(survey, response)
+        form_id = pck_transformer.get_cs_form_id()
+
+        self.assertEqual(form_id, '0020', msg=None)
```

Comment:
are these actually "testing" anything?

Reply:
A simple test to check to see if the qcas form types are valid?